docs/data/routes/docs/nextjs/tracking-and-analytics/configuration/en.md

```diff
@@ -0,0 +1,66 @@
+---
+name: configuration
+routeTemplate: ./data/component-templates/article.yml
+title: Enabling tracking and analytics
+---
+# Walkthrough: Enabling tracking and analytics
+
+This topic will guide you through the steps required to enable full Sitecore tracking and analytics for your Next.js application.
```

Comment:
Could we include a link to Sitecore docs where devs can learn more about the tracking & analytics functionality? (Assuming there is such a page...)

This page is very procedural, so it informs me how to enable tracking, but it's missing the why.

Edit: after seeing the next file in your PR, I now understand where the "why" is. I'm leaving the original comment so that @AncaIO can consider my original confusion when doing the reorg. Maybe we need a small excerpt of concepts on the procedural pages?

Reply:
> Could we include a link to Sitecore docs where devs can learn more about the tracking & analytics functionality? (Assuming there is such a page...)
> 
> This page is very procedural, so it informs me how to enable tracking, but it's missing the why.
> 
> Edit: after seeing the next file in your PR, I now understand where the "why" is. I'm leaving the original comment so that @AncaIO can consider my original confusion when doing the reorg. Maybe we need a small excerpt of concepts on the procedural pages?

I found this: https://doc.sitecore.com/developers/100/sitecore-experience-platform/en/web-tracking.html. Is this what you had in mind? 

After release, the link becomes https://doc.sitecore.com/developers/100/sitecore-experience-platform/en/web-tracking.html.

Reply:
@AncaIO That's the one. I've added links to this and other Sitecore XP/XM pages throughout this doc where appropriate.